poll_manager.cpp

```diff
@@ -509,20 +508,18 @@ void PollManager::pollTimerFired()
                                 }
 
                                 NodeValue &val = restNode->getZclValue(clusterId, attrId);
+                                quint16 maxInterval = val.maxInterval > 0 && val.maxInterval < 65535 ? (val.maxInterval * 3 / 2) : reportWaitTime;
 
+                                // This lines mean IKEA is never polled (or just once)?
                                 if (lightNode && lightNode->manufacturerCode() == VENDOR_IKEA && val.timestamp.isValid())
                                 {
                                     fresh++; // rely on reporting for ikea lights
                                 }
```

Comment:
Hmm strange I could swear that for IKEA lights polling was used until reporting kicks in.
I think it should be safe to remove this IKEA specific handling and just use the general approach.

Reply:
If memory serves, you did something special for IKEA, since the bulb firmware would hang when polled too often.  Not sure if this has been fixed in recent firmware versions.

Reply:
I pretty much had the similar discussion with Manuel on the IKEA stuff, but for reporting. There, it is explicitly prevented after the first report dripped in. Not sure if the mentioned side effects also occur on polling.

Reply:
It's just a gut feeling but the latest IKEA firmware versions appear more stable, haven't experienced issues in the last two months.

Reply:
So, gone they are...